service/storage_proxy.cc

```diff
@@ -6029,12 +6029,16 @@ storage_proxy::query(schema_ptr s,
 }
 
 future<result<storage_proxy::coordinator_query_result>>
-storage_proxy::query_result(schema_ptr s,
-    lw_shared_ptr<query::read_command> cmd,
+storage_proxy::query_result(schema_ptr s1,
```

Comment:
Suggest to rename the parameter query_schema to make it abundantly clear what has been transformed.

Reply:
(in .hh too, and the other callees)